### PR TITLE
Modified issue types to support content/site issues

### PR DESCRIPTION
**Pull request by austin-schaefer**

### CODE REVIEW

**Review by zstix — APPROVED** [review]

**Comment by austin-schaefer** [comment]
Hey @jpvajda @zstix , this is the PR to add issue types for docs issues along with a rename of the existing site-based issues. Note that the labels on the site-based ones have changed, so we'll probably want to relabel existing issues.

File: .github/ISSUE_TEMPLATE/content-enhancement.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+labels: content-enhancement
+assignees: ''
+---
+
+<!--
+Before opening a new issue, please search existing issues: https://github.com/newrelic/docs-website/issues.
+
+DISCLAIMER: Not every feature request will be worked on, but hearing about what you want is important. Don't be afraid to add a feature request! -->
+
+### Is your enhnacement request related to a problem? Please describe
```
Comment (jpvajda) [inline .github/ISSUE_TEMPLATE/content-enhancement.md:14]:
```suggestion
### Is your enhancement request related to a problem? Please describe
```

File: .github/ISSUE_TEMPLATE/content-bug.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+<!-- Please fill out each section below, otherwise, your issue will be closed. This info allows our writers to diagnose (and fix!) your issue as quickly as possible.
+
+** Check for existing issues**
+
+ Before opening a new issue, please search existing issues: https://github.com/newrelic/docs-website/issues -->
+
+### Description
+
+A clear and concise description of what the bug is.
```
Comment (jpvajda) [inline .github/ISSUE_TEMPLATE/content-bug.md:17]:
```suggestion
A clear and concise description of what the content bug is.
```

**Review by jpvajda — COMMENTED** [review]
Left some suggestions

**Comment by jpvajda** [comment]
@austin-schaefer I've changed the labels to math the ones you wanted. They should auto-update.

**Review by jpvajda — APPROVED** [review]
LGTM

**Pull request merged**